backend/src/middlewares/validateExpenseAuthority.js

```diff
@@ -0,0 +1,32 @@
+const { getRawFundingAgreementById } = require('../components/fundingAgreements')
+const log = require('../components/logger')
+
+/**
+ * Validates that the user is an Expense Authority for the FA's facility.
+ * @param
+ * @returns
+ */
+module.exports = function () {
+  return async function (req, res, next) {
+    log.verbose(`validating expense authority`)
+
+    const contactId = req?.body.contactId
+    // Only validate a signing update (which will include contactId)
+    if (!contactId) next()
```

Comment:
If there's no contactId, do we maybe want to return `next()` here to stop the flow early?

Reply:
Yes. This was missed.